logstash-core/src/main/java/org/logstash/ackedqueue/Queue.java

```diff
@@ -106,6 +106,30 @@ public Queue(String dirPath, int pageCapacity, long maxBytes, CheckpointIO check
         }
     }
 
+    public String getDirPath() {
+        return dirPath;
```

Comment:
ok thanks - please do the others as well, I did not comment on each one specifically. Thanks!